docker-compose.yml

```
@@ -0,0 +1,56 @@
+version: "3"
+
+services:
+  redis:
+    image: "redis:7-alpine"
+    hostname: "envless-redis"
+    init: true
+    networks:
+      - envless_network
+    ports: 
+      - "6379:6379"
+
+  redis-rest:
+    image: "nicolas/webdis:0.1.21"
+    platform: linux/amd64
+    hostname: "envless-redis-rest"
+    depends_on:
+      - "redis"
+    ports: 
+      - "7379:7379"
+
+  postgres:
+    image: "postgres:15.2-alpine"
+    hostname: "envless-postgres"
+    restart: "always"
+    environment:
+        POSTGRES_USER: "envless"
+        POSTGRES_DB: "envless"
+        POSTGRES_PASSWORD: "envless"
+    volumes:
+      - postgres_data:/var/lib/postgresql/data
+    networks:
+      - envless_network
+    ports: 
+      - "5433:5433"
+
+  platform:
+    container_name: platform
+    build:
+      context: .
+      dockerfile: Dockerfile
+    restart: always
+    env_file: .env.example
+    ports:
+      - 3000:3000
```

```suggestion
      - 3000:3000
      - 3883:3883
```

We also need to expose port `3883:3883` (currently used by email interceptor)

---

Got it.